content/en/docs/_index.md

```diff
@@ -1,16 +1,28 @@
 ---
-title: "Documentation"
+title: "Welcome to Axway AMPLIFY Shared Services documentation"
 linkTitle: "Documentation"
 weight: 20
+no_list: true
 menu:
   main:
     weight: 20
 ---
 
-{{% pageinfo %}}
-This is a placeholder page that shows you how to use this template site.
-{{% /pageinfo %}}
+Axway AMPLIFY Shared Services documentation is available for collaboration using an open source model as part of the Axway Open Documentation Open Beta program.
 
-This section is where the user documentation for your project lives - all the information your users need to understand and successfully use your project.
+## About the Open Beta program
 
-For large documentation sets we recommend adding content under the headings in this section, though if some or all of them don’t apply to your project feel free to remove them or add your own.
\ No newline at end of file
+The Open Beta enables you to suggest edits and create issues for any page of the documentation using a GitHub account. To learn more, see the [contribution guidelines](/docs/contribution_guidelines/).
+
+The role of each documentation website in the Open Beta is as follows:
+
+* Axway Open Documentation on Netlify (<https://axway-open-docs.netlify.app/>) - Specific product documentation sets are available on this website for collaboration using GitHub and Netlify CMS.
```

Comment:
on this website for **contribution**  ?? I think 'collaboration' is ok on the fist paragraph, gives a sense of  whole, but here where it's more specific, 'contribution' reads better IMHO

Reply:
agree thanks will change